### DISCUSSION TITLE
Cannot find in HACS

### DISCUSSION THREAD

**Discussion by Kurisudes** [dckiller51/bodymiscale#47] [Q&A]
Hello,
This add-on looks very nice. I would love to get it but it does not appear when I am trying to add it from HACS.
Am I doing something wrong?
![Screen Shot 2021-12-30 at 0 04 39](https://user-images.githubusercontent.com/96847661/147709645-78a12b37-bdcf-4673-b49b-8898cacf22ef.png)

**Comment by dckiller51** [comment]
Hello, you are in the add-ons community and not in Hacs. Did you find?
